src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,205 @@
+@mixin onMobile {
+  @media (max-width: 576px) {
+    @content;
+
+    .header {
```

Comment:
you use these mixins a bit unproper

how should it look like:
you have mixin with defined `@media` and with `@content`

then, if you want to add some style to the header, that will be applied only for mobile phones you should do following
```
.header {
  // some styles
  
  @include onMobile {
    // styles you want to be applied for mobile phones
  }
```

Reply:
Did I understand correctly?

@include → I use only in the selectors and not @media!  right?
@media → I'm setting a rule, and styles already in the @include.

But! how to be with duplicated @include in the selectors? 

Thanks for the answer.